src/mds/events/EExport.h

```diff
@@ -35,7 +35,7 @@ class EExport : public LogEvent {
   EExport() :
     LogEvent(EVENT_EXPORT), target(MDS_RANK_NONE) { }
   EExport(MDLog *mdlog, CDir *dir, mds_rank_t t) :
```

Comment:
@batrick Does argument 'MDLog *mdlog' not required removal?

Reply:
It probably does but for now I'm leaving it there.